modules/ossm-performance-scalability.adoc

```diff
@@ -0,0 +1,79 @@
+////
+This module included in the following assemblies:
+- /v2x/customizing-installation-ossm.adoc
+////
+
+[id="ossm-performance-scalability_{context}"]
+= Performance and scalability
+
+{ProductName} strives to provide features with minimal resource overhead and aims to support very large meshes with high request rates while adding minimal latency.
+
+== Performance summary for Istio 1.6.8
+
+The upstream Istio community load tests mesh consists of *1000* services and *2000* sidecars with 70,000 mesh-wide requests per second.
```

Comment:
As a continuation to my comment about pointing out the istio version, can we say here, that the istio team did these tests, but they are relevant for this product? I think it might be okay to say running the tests in istio 1.6.8 in the line below if we were to be more clear that we're asserting that it's true about this product, even though we got the information from Istio.